[2024-11-19T09:17:04.714+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_data.generate_fraud_data manual__2024-11-19T09:17:01.904877+00:00 [queued]>
[2024-11-19T09:17:04.725+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_data.generate_fraud_data manual__2024-11-19T09:17:01.904877+00:00 [queued]>
[2024-11-19T09:17:04.726+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-11-19T09:17:04.741+0000] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): generate_fraud_data> on 2024-11-19 09:17:01.904877+00:00
[2024-11-19T09:17:04.753+0000] {standard_task_runner.py:60} INFO - Started process 3898 to run task
[2024-11-19T09:17:04.756+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'generate_data', 'generate_fraud_data', 'manual__2024-11-19T09:17:01.904877+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmpc933zk0v']
[2024-11-19T09:17:04.758+0000] {standard_task_runner.py:88} INFO - Job 65: Subtask generate_fraud_data
[2024-11-19T09:17:04.799+0000] {task_command.py:423} INFO - Running <TaskInstance: generate_data.generate_fraud_data manual__2024-11-19T09:17:01.904877+00:00 [running]> on host 4cf1ca1acf00
[2024-11-19T09:17:04.881+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='generate_data' AIRFLOW_CTX_TASK_ID='generate_fraud_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-19T09:17:01.904877+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-19T09:17:01.904877+00:00'
[2024-11-19T09:17:04.968+0000] {logging_mixin.py:188} INFO - Error inserting labeled transaction data: 1062 (23000): Duplicate entry '392' for key 'labeled_transactions.PRIMARY'
[2024-11-19T09:17:04.970+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-11-19T09:17:04.979+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=generate_data, task_id=generate_fraud_data, execution_date=20241119T091701, start_date=20241119T091704, end_date=20241119T091704
[2024-11-19T09:17:05.009+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-19T09:17:05.024+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
